### ISSUE TITLE
[Feature Request] Proxy service request and response without (de)serialization

### ISSUE THREAD

**Issue by fujitatomoya** [Barry-Xu-2018/ros2_load_balancing_service#5]
### Description

This requires the new feature in `rclcpp`, `rcl` and `rmw` such as `rmw_take_serialized_request`.

Currently in the code load balancer does (de)serialization when it needs to proxy the service request and response messages.
This is what we can do at this moment, but it would be much better for the performance and efficiency to proxy the serialized data without (de)serialization in the load balancer.

**Comment by Barry-Xu-2018** [comment]
Yes, the current forwarding efficiency is quite low because it doesn't directly forward serialized data.  
Like with Topic, we need the RMW layer to provide an interface to directly access serialized data, which can then be encapsulated through rcl and rclcpp for use at higher levels.  

Once this new interface is ready, I will update code to provide better performance.
